src/Scheduler/CronDefinition.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Scheduler;
+
+use DateInterval;
+
+final class CronDefinition
```

Comment:
I don't really like this name. Either it should be `Schedule` or `Cron`, my preference is the first.